.prettierrc

```diff
@@ -2,5 +2,14 @@
   "arrowParens": "always",
   "singleQuote": true,
   "trailingComma": "es5",
-  "bracketSpacing": false
+  "bracketSpacing": false,
+  "endOfLine": "auto",
+  "overrides": [
```

Comment:
By naming our fixture file `check.inert.component.html` we should be able to trigger the angular parser without needing to add this custom override. Can you confirm if that works and if so rename the fixture file and remove this?